alfalfa_worker/lib/testcase.py

```diff
@@ -86,7 +86,7 @@ def __init__(self, **kwargs):
         self.u_store = copy.deepcopy(self.u)
         # Set default options
         self.options = self.fmu.simulate_options()
-        self.options['CVode_options']['rtol'] = 1e-6
+        # self.options['CVode_options']['rtol'] = 1e-6

```

Comment:
CVode no longer included by default. FMUs must provide solver.

Reply:
@TShapinsky @kbenne If we are changing expected format/requirements for FMU upload, we should update docs.

Reply:
Indeed FMUs for "co-simulation" are now the standard for BOPTEST and I guess here too and indeed that means the solver is baked into the FMU. 

I think this PR is key context.

https://github.com/ibpsa/project1-boptest/pull/536